.github/workflows/utils/log_parser.py

```diff
@@ -130,14 +130,14 @@ def _common_line_splitter(
             return line[(start_index
                         + len(text_to_split_start)):
                         end_index].strip()
-    return line[(start_index + len(text_to_split_start)):].strip()
+    return line[(start_index + len(text_to_split_start)):].split(None, 1)[0]
 
 
 def asan_line_splitter(
         line: str):
     return _common_line_splitter(
         line=line,
-        text_to_split_start='==ERROR: ')
+        text_to_split_start='==ERROR: AddressSanitizer: ')
```

Comment:
This leaves out some errors kinds like `LeakSanitizer` (see [here](https://github.com/eProsima/Fast-DDS/actions/runs/4199782572))

Reply:
Addressed in b601c28c1

Reply:
Addressed in b601c28